components/button/test/button-icon.visual-diff.js

```diff
@@ -31,20 +31,20 @@ describe('d2l-button-icon', function() {
 				it(name, async function() {
 
 					if (name === 'hover') {
-						if(entry.category === 'translucent-enabled') {
+						if (entry.category === 'translucent-enabled') {
```

Comment:
I think this is part of why I hadn't jumped right into updating the tests to this format.  As more and more conditional logic is added to accommodate different tests, the simplicity of the format become lost, and I'm not sure how much logic is too much and when to pull certain tests out on their own.

Reply:
In that case, should I revert these?

Reply:
I don't necessarily think so, or have a strong opinion on it.  Just not sure how far we take it.  In other components I can imagine some more complex logic.

Reply:
Haha yeah it is getting a little crazy eh. Another option here to reduce complexity would be to put CSS classes on the things that should be focused or hovered upon. For example, we could put `hover-target` and `focus-target` on the different bits. But yeah, complicated.